src/lib/platform/XWindowsScreen.h

```diff
@@ -227,8 +227,8 @@ class XWindowsScreen : public PlatformScreen {
     bool m_screensaverNotify;
 
     // logical to physical button mapping.  m_buttons[i] gives the
-    // physical button for logical button i+1.
     std::vector<unsigned char> m_buttons;
+    // physical button for logical button i.
```

Comment:
Seems like there was a merge conflict? The comment is now split across the declaration.

Reply:
Anyway, I fixed this issue and updated the branch.